consensus/test_data/build.sh

```diff
@@ -0,0 +1,55 @@
+#! /bin/bash
+
+cd $GOPATH/src/github.com/tendermint/tendermint
+
+# specify a dir to copy
+# NOTE: eventually we should replace with `tendermint init --test`
+DIR=$HOME/.tendermint_test/consensus_state_test
+
+# XXX: remove tendermint dir
+rm -rf $HOME/.tendermint
+cp -r $DIR $HOME/.tendermint
+
+function reset(){
+	rm -rf $HOME/.tendermint/data
+	tendermint unsafe_reset_priv_validator
+}
+
+reset
+
+# empty block
+tendermint node --proxy_app=dummy &> /dev/null &
+sleep 5
+killall tendermint
+
+sed '/HEIGHT: 2/Q' ~/.tendermint/data/cs.wal/wal  > consensus/test_data/empty_block.cswal
```

Comment:
/q would print up to and including the match, then quit. /Q doesn't include the match. I'm no sed guru, I just read SO: http://unix.stackexchange.com/questions/11305/grep-show-all-the-file-up-to-the-match